doc/src/manual/parallel-computing.md

```diff
@@ -1,10 +1,265 @@
 # Parallel Computing
 
-This part of the manual details the following types of parallel programming available in Julia.
+For any newcomer to multi-thread and parallel computing it can be useful to first appreciate 
+the different levels of parallelism offered by Julia. We can divide them in three main categories :
+
+1. Green Threading or Julia Coroutines 
```

Comment:
"Julia Coroutines (green threading)" might be better as coroutines is used in the paragraph below.